tests/test_voice_connection.py

```
@@ -0,0 +1,195 @@
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+from src.utils.voice_recorder import VoiceRecorder
+
+
+class TestVoiceRecorder:
+    """VoiceRecorder 測試類別"""
+
+    def setup_method(self):
+        """每個測試方法前的設置"""
+        self.voice_recorder = VoiceRecorder()
```

Pylint (3.3.7)

[warning] Attribute 'voice_recorder' defined outside __init__

(W0201)